src/game/playerShooting.js

```diff
@@ -0,0 +1,39 @@
+// import { computerState, computerShips, computerEmptyFields } from "./game";
```

Comment:
If something is commented that it is unneeded - remove it

Reply:
:+1: